src/countervalues/mock.js

```diff
@@ -32,3 +33,93 @@ export const genStoreState = (pairs: Pair[]): CounterValuesState => {
   });
   return state;
 };
+
+export const getDailyRatesImplementation = (
+  getAPIBaseURL: () => string,
+  pairs: PollAPIPair[]
+) => {
+  const rates = {
+    USD: { BTC: { SIMEX: { latest: 0.00521587214628 } } },
+    BTC: {}
+  };
+
+  // For pairs not reflected here we will be falling back on 1  ¯\_(ツ)_/¯
```

Comment:
I would suggest instead to fallback to no countervalues? because we want to test the "partial" aspect & we'll have it when doing erc20 ;)

for the record, this is a script that i've used to generate that list: https://github.com/LedgerHQ/ledger-live-common/blob/master/scripts/listMarketPriceFor1USD.js we might adapt in future to add more tokens ;) i think we have listTokens() already

Reply:
I'll look into it later, looks good

Reply:
ok, zero it is

Reply:
ok thanks! i'll later re-generate that rates list and probably having it located at a same place & a script to refresh it . but not blocking this PR , we'll only need it for ERC20 mocking